### PR TITLE
Revert dropped support to old pytest versions

### PR DESCRIPTION
**Pull request by nicoddemus**
This release reverts the change that dropped support for older pytest versions. We should consider this change only for `1.23`.

Issues:

https://github.com/pytest-dev/pytest/issues/3724
https://github.com/pytest-dev/pytest-xdist/issues/319

### CODE REVIEW

**Comment by nicoddemus** [comment]
Oh of course. It was because of `six`.

So to fix this we would need to either revert `six` usage, or include `six` as a dependency. The first is not good because it will generate more work for us (remove and include it back in a future release), the second will likely cause problems as well.

What do you suggest @RonnyPfannschmidt?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,16 @@
+pytest-xdist 1.22.5 (2018-07-27)
+================================
+
+Bug Fixes
```
Comment (RonnyPfannschmidt) [inline CHANGELOG.rst]: there seems to be a `B` missing

Comment (RonnyPfannschmidt) [reply CHANGELOG.rst]: i suggest to do a minor release adding six

Comment (nicoddemus) [reply CHANGELOG.rst]: OK, done.

About the missing `B`, it is some bug on GH's diff, [the file is fine](https://github.com/nicoddemus/pytest-xdist/blob/release-1.22.5/CHANGELOG.rst).

**Pull request merged**